src/components/AuthButton/AuthButton.jsx

```diff
@@ -22,7 +23,9 @@ const AuthButton = () => {
       </Button>
     </p>
   ) : (
-    <p>You are not logged in.</p>
+    <Typography textAlign="center" mt={3}>
+      You are not logged in.
```

Comment:
:wrench: This needs intl.

Reply:
Addressed:  953fa234eb8cfde9d37713d1f2ca443865e88876